plugins/catalog/src/components/AboutCard/AboutCard.tsx

```diff
@@ -88,15 +91,16 @@ export interface AboutCardProps {
 /**
  * Exported publicly via the EntityAboutCard
  */
-export function AboutCard(props: AboutCardProps) {
-  const { variant } = props;
+export function AboutCard({ variant }: AboutCardProps) {
```

Comment:
Please keep unpacking this inside the component body; we explicitly want it that way because if you do it like this, defaults etc become part of the externally visible contract and API reports which is ugly.

Reply:
First time I see anything about API Reports in a project, makes sense, will revert this.